fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs

```diff
@@ -618,8 +612,7 @@ impl A380SpoilerFactory {
         let spoiler_3 = Self::new_a380_spoiler_element(context, id, 3, None);
         let spoiler_4 = Self::new_a380_spoiler_element(context, id, 4, None);
         let spoiler_5 = Self::new_a380_spoiler_element(context, id, 5, None);
-        let spoiler_6 =
-            Self::new_a380_spoiler_element(context, id, 6, Some(Self::SPOILER_6_EBHA_BUS));
+        let spoiler_6 = Self::new_a380_spoiler_element(context, id, 6, Some(AC_EHA_BUS));
```

Comment:
Same goes for the spoiler 6. Powered by the AC ESS bus.

Reply:
Mmmm went lost with test ref I used :(.